docker/docker-compose.yml

```diff
@@ -4,47 +4,47 @@ services:
   db:
     image: postgres:9.5.3
     volumes:
-      - $VAGRANT_DATA_DIR../data/pgdata:/var/lib/postgresql/data/pgdata
-    environment:
-      PGDATA: /var/lib/postgresql/data/pgdata
-      POSTGRES_USER: listenbrainz
-      POSTGRES_PASSWD: listenbrainz
-      POSTGRES_DB: listenbrainz
+      - ./data/pgdata:/var/lib/postgresql
```

Comment:
I believe that the postgres image exposes `/var/lib/postgresql/data/` as a volume, so we should use this instead of `/v/l/postgres`.

I don't like this way of persisting data with postgres because the local directory ends up owned by the `postgres` user, which makes it difficult to `rm`.
Instead, I prefer to use a named volume, which is stored in `/var/lib/docker` and can be managed using the `docker volumes` command. See https://github.com/AudioCommons/ac-mediator/blob/master/docker-compose.yml#L2-L3 for an example.